### ISSUE TITLE
PAC3 Entity/Bones not showing correctly when other players come back in range unless re-worn

### ISSUE THREAD

**Issue by FrostinePAC3** [CapsAdmin/pac3#1135]
Hello! for some time now it seems like I and a few other players have noticed a problem with either the **Entity** part, **bones** (or a combined mix) Upon wearing a PAC _Ex. (a mouse PAC)_ 
![image](https://user-images.githubusercontent.com/96437971/146806338-b2f17f6b-169e-4cce-a9fc-e28e09dab50b.png)
  It's normal to everyone, but if they walk away out of range and come back, suddenly it's as if the entity didn't exist at all and it stretches or shrinks the size/bones:  _(Size is supposed to be 0.2 but it shows as 1)_
![image](https://user-images.githubusercontent.com/96437971/146806525-d8ec2c6a-6a3c-4ccf-b807-82f95f8a6b6e.png)
**(Disclaimer, Client-side the PAC looks normal. To everyone else it breaks UNLESS Re-worn)** 
Is there any cause for this issue? or Is there a command I can implement into PACs that 'reload' them? 
I've been making these 'sphere' PACs for years and have never had this issue 'till now.
Thanks again in advance and apologies if this question is annoying.

**Comment by thegrb93** [comment]
Is it the 'entity' 'size' modifier having the problem? What do you mean by range, the pac3 render distance limit?

**Comment by FrostinePAC3** [comment]
> Is it the 'entity' 'size' modifier having the problem? What do you mean by range, the pac3 render distance limit?

Yes, range refers to the pac3 render distance, in short- people coming in range of me will see my pac stretched out until I rewear it. 
And I suspect it might be something with the Entity part since bones don't seem 'out of place' it's mostly 'size' contributing to the stretched/exploded effect. (Also I have tried hiding and un-hiding the 'bones' and 'entity' with an event to see if that would temporarily fix the issue but it does not.

**Comment by thegrb93** [comment]
What version is being used? Does an earlier version fix it?

**Comment by thegrb93** [comment]
I think the pac_version console command might help with getting the version

**Comment by FrostinePAC3** [comment]
> I think the pac_version console command might help with getting the version
Okay, so inputting it in console gives me this (checked 3 times just in case) : 
] pac_version
[PAC3] Addon = Entity-About-Set / 1c6e54f37aa54d061e743afdf1025cf
[PAC3] Core = Vertical-Post-Delay / 4b1711ec8e5d3896f113ae138fdb684d
[PAC3] Editor = Cur-Physical-Table / a2b0b3f29f239ec9d1c7c6155cbbfb92

**Comment by thegrb93** [comment]
Ok, I was wrong, that command is completely useless. Do you know if the server is using develop or workshop?

**Comment by FrostinePAC3** [comment]
> Ok, I was wrong, that command is completely useless. Do you know if the server is using develop/master/workshop? Would they be willing to try develop to see if that resolves the issue?

I could definitely ask the next time a dev or higher-up on the server shows up!

**Comment by FrostinePAC3** [comment]
> > Ok, I was wrong, that command is completely useless. Do you know if the server is using develop/master/workshop? Would they be willing to try develop to see if that resolves the issue?
> 
> I could definitely ask the next time a dev or higher-up on the server shows up!

Okay! finally something, we're currently using the latest master branch on here if that helps. 
And it seems we're not using the dev branch right now since the flexes are a bit janky on them atm?

**Comment by thegrb93** [comment]
I've never heard of flex jank. They should report those if so.

**Comment by thegrb93** [comment]
Has the latest pac version helped?

**Comment by FrostinePAC3** [comment]
> Has the latest pac version helped?

As far as I know, this seems to be a recurring issue/conflict with ppm2 and its entity/scaling system. Basically two conflicting entities

**Comment by thegrb93** [comment]
Do you have a link to that addon?

**Comment by FrostinePAC3** [comment]
> Do you have a link to that addon?

I'm pretty sure it's the one dbot worked on 
https://steamcommunity.com/sharedfiles/filedetails/?id=933203381
As far as I know this wasn't much of a problem before but I could be wrong

**Comment by thegrb93** [comment]
Any idea if this is a ppm conflict, @DBotThePony ?
